.travis.yml

```diff
@@ -5,22 +5,28 @@ dist: trusty
 
 python:
   - "3.6"
-  - "3.6-dev"
-  - "3.7-dev"
+  - "3.6.5"
+  - "3.7.2"
   - "nightly"
 
 matrix:
   allow_failures:
-    - python: 3.6-dev
-    - python: 3.7-dev
+    - python: 3.6.5
+    - python: 3.7.2
     - python: nightly
 
+before_install:
+  - pip install poetry
+
 install:
-  - pip install --process-dependency-links .[dev]
-  - pip install codacy-coverage
+  - poetry install --no-dev --extras ci
```

Comment:
We want to install it without the dev dependencies? Wouldn't we want to install it with both dev and CI deps so we'd be able to run automated tests and stuff?